ci/download-core-sdk.sh

```diff
@@ -35,6 +35,7 @@ spatial package retrieve "worker_sdk" "core-dynamic-x86-win32"       "${PINNED_C
 spatial package retrieve "worker_sdk" "core-dynamic-x86_64-win32"    "${PINNED_CORE_SDK_VERSION}" "${CORE_SDK_DIR}/worker_sdk/core-dynamic-x86_64-win32"
 spatial package retrieve "worker_sdk" "core-dynamic-x86_64-linux"    "${PINNED_CORE_SDK_VERSION}" "${CORE_SDK_DIR}/worker_sdk/core-dynamic-x86_64-linux"
 spatial package retrieve "worker_sdk" "core-bundle-x86_64-macos"     "${PINNED_CORE_SDK_VERSION}" "${CORE_SDK_DIR}/worker_sdk/core-bundle-x86_64-macos"
+spatial package retrieve "worker_sdk" "core-dynamic-arm-ios"         "${PINNED_CORE_SDK_VERSION}" "${CORE_SDK_DIR}/worker_sdk/core-dynamic-arm-ios"
```

Comment:
there should also be one for x86 / emulator

Reply:
Yeah, we'll need it for iOS simulator, but I think getting it working will need a few more things in Unity.
I was planning on creating a separate diff to address that.
Does that make sense?

Reply:
I don't think that there is much more needed on the Unity side, it should just work. But I'm fine with having it in a separate PR